collector/exporter.go

```diff
@@ -37,6 +37,10 @@ var (
 		"exporter.log_slow_filter",
 		"Add a log_slow_filter to avoid slow query logging of scrapes. NOTE: Not supported by Oracle MySQL.",
 	).Default("false").Bool()
+	tagQueries = kingpin.Flag(
+		"exporter.tag-queries",
+		"TODO",
+	).Default("false").Bool()
```

Comment:
I don't think we need a flag for this, let's just add the feature so it's always on. :smile:

Reply:
This flag should allow the enduser to pass N number of tags. Tags should not be static, and should be a Key/Value pair so that monitoring systems (such as Grafana) can filter on the tag and value.